src/click/termui.py

```diff
@@ -137,6 +141,18 @@ def prompt_func(text):
         text, prompt_suffix, show_default, default, show_choices, type
     )
 
+    if confirmation_prompt:
+        confirmation_prompt = _build_prompt(
```

Comment:
`_build_prompt` is for adding information about the type and default value, which isn't relevant when confirming.